app/javascript/packs/braintree_edit_form.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+// TODO: we can probably re-use BraintreeForm here
+/* eslint-disable no-unused-expressions */
+
+import braintree from 'braintree-web'
+import { createBraintreeClient, createHostedFieldsInstance, hostedFieldOptions } from 'PaymentGateways/braintree/braintree'
+import { error } from 'utilities/alert'
+
+document.addEventListener('DOMContentLoaded', async () => {
+  const submit = document.querySelector('button[type="submit"]')
+  const data = document.querySelector('#braintree_data')
+  const form = document.querySelector('form#new_customer')
+
+  if (!submit || !data || !form) {
+    throw new Error('Required Element not found')
+  }
+  const authorization = data.dataset?.authorization || ''
+
+  const client = await createBraintreeClient(braintree.client, authorization)
+  const instance = await createHostedFieldsInstance(braintree.hostedFields, client, hostedFieldOptions)
+
+  submit.removeAttribute('disabled')
+  form.addEventListener('submit', (event: Event) => {
+    event.preventDefault()
+    event.stopPropagation()
+    instance.tokenize((tokenizeErr, payload) => {
+      if (tokenizeErr) {
+        error('Some of the credit card details are wrong. Please update them.')
```

Comment:
We're setting a fixed error message in here instead of using `tokenizeErr`, as in the past. Why is that?

Reply:
`tokenizeErr` is an object containing info too cryptic for users. On the other hand, I opted for showing an alert in the UI because right now the page fails without any kinds of feedback. While playing with it I realized there's only 2 errors Braintree will return:
* All your fields are empty
* Some of your fields are wrong
But of course it only refers to the credit card fields, which are the ones being handled by Braintree. Simply printing the "all your fields are wrong" built-in message will be very confusing for the user.